lib/fathead/python/cover/functions.txt

```diff
@@ -106,3 +106,18 @@ def
 for
 lambda
 try
+csv read, csv.reader
+enum list, enumerate
+call process, subprocess
+__import__, __import__
+datetime.datetime, datetime
+datetime tzinfo, datetime.datetime.timetz
+datetime to timestamp, datetime.datetime.timestamp
+datetime tzinfo, datetime.datetime.timetz
+format string, string.Formatter.format
+glob, glob
```

Comment:
this can be removed, we don't need redirects that are identical to the existing article titles 👍